smart-contracts/LimitUpdating.md

```diff
@@ -2,9 +2,11 @@
 
 If you are trying to whitelist and update token limits many addresses at once, you will need to use the bulkSetTokenLockBurnLimit.js script. If you need to only update the lock or burn limit and the token has already been whitelisted, use the setTokenLockBurnLimit script.
 
-To bulk update the whitelist and limits for each token, use bulkSetTokenLockBurnLimit.js like so:
+Before running the following script go to the data folder and open the `limitWhitelistUpdate.json`. Edit the address value to the address you want to whitelist. Then, update the limit to the desired limit you would like to allow a user to move through the bridge in a single tx. If you want to set this to the max allowable value in solidity, set the limit value to `115792089237316195423570985008687907853269984665640564039457584007913129639935` which is the UINT_MAX in solidity.
```

Comment:
can we change this to suggest the same process as we're going with now, ie:
 - create a new whitelist_network_date.json file
 - make a PR
 - get a review/approval
 - get it manually deployed